modules/mongodb/mongodb.go

```diff
@@ -208,3 +222,7 @@ func withAuthReplicaset(
 		return nil
 	}
 }
+
+func waitForMongoReady(ctx context.Context, c testcontainers.Container, cli mongoCli) error {
```

Comment:
BTW, I noticed that this function is unused now. Could you please remove it, or verify it's not needed? 🙏